### PR TITLE
[onert] Add shape inference for strided_slice operation

### PR DESCRIPTION
**Pull request by intom**
Shape Inference for STRIEDED_SLICE operation

ONE-DCO-1.0-Signed-off-by: sungho22.lee <sungho22.lee@samsung.com>

### CODE REVIEW

File: runtime/onert/core/src/util/shapeinf/StridedSlice.cc [hunk 1]
```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
```
Comment (seanshpark) [inline runtime/onert/core/src/util/shapeinf/StridedSlice.cc:3]: If some portion of the code came from TensorFlow, please add copyright from it.

Comment (intom) [reply runtime/onert/core/src/util/shapeinf/StridedSlice.cc:4]: I added copyright.

**Comment by hyunsik-yoon** [comment]
could you add one of a contributor in WG2? Thanks!

File: runtime/onert/core/src/util/shapeinf/StridedSlice.cc [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+}
+
+void DynamicInferer::visit(const ir::operation::StridedSlice &op)
+{
+
+  const auto input_index{op.getInputs().at(ir::operation::StridedSlice::Input::INPUT)};
+  auto input = _tensor_registry->getITensor(input_index);
+  ir::Shape input_shape = getShape(input.get());
+
+  if (!input->is_dynamic())
```
Comment (hyunsik-yoon) [inline runtime/onert/core/src/util/shapeinf/StridedSlice.cc:268]: If `input` was static but, e.g., `ends` was an output of an op, `output` is dynamic (set by static tensor inferer).
However in this method, dynamic shape inferer does simply returns without allocating the output memory.

**Comment by glistening** [comment]
@intom Please resolve conflict.

**Pull request merged**